examples/laksaj-examples/ljtest/src/test/java/com/zilliqa/ljtest/AppTest.java

```diff
@@ -0,0 +1,20 @@
+package com.zilliqa.ljtest;
+
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+/**
+ * Unit test for simple App.
+ */
+public class AppTest 
+{
+    /**
+     * Rigorous Test :-)
+     */
+    @Test
+    public void shouldAnswerWithTrue()
+    {
+        assertTrue( true );
```

Comment:
It would appear that this test does not serve much purpose - is this on purpose?

Reply:
Ah - sorry - I'll add a comment. Its purpose is to prove that you can fetch the jar file from Maven Central.